### ISSUE TITLE
'Unnecessary semicolon' when multiline code.

### ISSUE THREAD

**Issue by jeemin1** [palantir/tslint#3137]
### Bug Report

- __TSLint version__:5.6.0
- __TypeScript version__:2.4.2
- __Running TSLint via__: (pick one) Node.js API
- WebStorm 2017.2.2

#### TypeScript code being linted

```ts
export class Test {
    // ok
    public test1 = () => new Promise((rtn, rej) => rtn()).then(console.log);

    // Unnecessary semicolon
    public test2 = () => new Promise((rtn, rej) => rtn())
        .then(console.log);

    // Unnecessary semicolon
    public test3 = () => new Promise((rtn, rej) => rtn())
        .then(console.log)
    ;
}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "semicolon": [
      true,
      "always"
    ]
  }
}
```

#### Actual behavior
'test1' is ok. but not others. Why? Those are only different the line breaks.

#### Expected behavior
I think all three should have the same reaction.

**Comment by ajafff** [comment]
Class properties that are initialized with a **multiline** arrow function (called "bound class methods") are by default treated like methods. That means there is no semicolon allowed.
That's a bit opinionated. You can disable that behavior with `"ignore-bound-class-methods"` option.

**Comment by jeemin1** [comment]
Thank you! I did not know it.
